### PR TITLE
`redis` Add `jGet` function

### PR DESCRIPTION
**Pull request by mtuchi**
## Summary

Add a  `jGet(key: string)`. Ref #714 

**Notes**
changeset is in `release/redis` branch

## Review Checklist

Before merging, the reviewer should check the following items:

- [x] Does the PR do what it claims to do?
- [x] Are there any unit tests? Should there be?
- [ ] Is there a changeset associated with this PR? Should there be? Note that
      dev only changes don't need a changeset.

### CODE REVIEW

File: packages/redis/src/Utils.js [hunk 1]
```diff
@@ -48,3 +48,11 @@ export const assertSetArgs = (key, value) => {
     `Make sure both key and value are strings for set(): e.g., set('name', 'fela')`
   );
 };
+
+export const assertjGetArgs = key => {
+  assertArgType(
+    key,
+    'string',
+    `Make sure to pass a strings for jGet(): e.g., jGet('patient').`
```
Comment (josephjclark) [inline packages/redis/src/Utils.js:56]: `pass a string` (string singular) would be better english. And don't end error/status/console messages in full stops please.

**Review by josephjclark — CHANGES_REQUESTED** [review]
The get and set could have been in the same PR as they're very closely related functions. Also there's one issue for both functions.

But it's OK.

A changeset is needed here

**Comment by josephjclark** [comment]
Oh, wait, the changeset is on the release branch? Why?

**Comment by mtuchi** [comment]
>Oh, wait, the changeset is on the release branch? Why?

I wanted to have one minor version bump and not multiple changeset which will create multiple minor version bump

File: packages/redis/src/Utils.js (hunk 1 above)
Comment (mtuchi) [reply packages/redis/src/Utils.js]: Fixed the typo

**Comment by josephjclark** [comment]
@mtuchi I don't understand

If you release 3 minor changes at once, changeset will generate one minor version bump. That's the whole point of it, that's why we're using it.

By commiting the changeset to the release branch, you are promising that the release branch will evenually contain that feature. But it doesn't.

What if I decide to release the scan function today and these new functions next week? The release branch is currently in an inconsistent state and we all have to hope that I notice or remember that there are unimplemented changes.

Please keeps things simple. 

Whenever making a change that the user needs to be informed of, generate a changeset on that branch.

**Review by josephjclark — APPROVED** [review]

**Comment by mtuchi** [comment]
@josephjclark thanks for the clarity, I think my thinking process hard some flaw 😞, I didn't want to create multiple versions so i thought the changeset should be in the release branch. I think i understand more now how to use the changeset, thanks for the clarification 🙌🏽

**Pull request merged**